README.md

```diff
@@ -60,34 +69,53 @@ make -C ${ZKLLVM_BUILD:-build} assigner clang -j$(nproc)
 
 zkLLVM's workflow is as follows:
 
-1. Users wishing to prove some statement are supposed to implement an application in a language compatible with some frontend (C++ for now). This code will be compiled with a modified version of the `clang` compiler, which will output intermediate representation of the circuit. 
-   > For the most performant cryptography circuits (e.g. hashes, signatures, VDFs, proof system verifications, etc.) 
-   > we recommend using [=nil; Foundation's Crypto3 library](https://github.com/nilfoundation/crypto3.git).
+> TODO: are statement and circuit the same thing? 
+> Why do we use both 'prove a statement' and
+> 'generate a proof of the circuit'?
+
+1.  Users wishing to prove a statement should implement an application in a language
```

Comment:
This section has a few things mixed up:

* The overall usage flow for the users who need proofs calculated: write an app in a compatible language, compile the app, ..., post an order, and get the proof.
* Technical details of a proof order: how to compile, what files are there, what the proof-generators will get from them.
* There's a proof market and it's owned by =nil;
* There's also a nice cryptography library which we recommend.

All this should be split. The usage flow goes up, technical details get their documentation links.